commit ebd8f9b3fd0f96239b2476e9669621e1ee7899ba
Author: davidw5 <37127909+davidw5@users.noreply.github.com>
Date:   Sat Mar 10 01:17:17 2018 +1100

    Update arm64 Makefile to compile bcm2710 dtb file
    
    The line 'dtb-$(CONFIG_ARCH_BCM2835) += bcm2710-rpi-3-b.dtb' has been copied from previous rpi-4.14.y version into rpi-4.15.y arch/arm64/boot/dts/broadcom/Makefile to restore compilation of bcm2710-rpi-3-b.dtb device tree blob under 'make ARCH=arm64 dtbs' command.

diff --git a/arch/arm64/boot/dts/broadcom/Makefile b/arch/arm64/boot/dts/broadcom/Makefile
index 82c54819a6c7..9085d5ba46a0 100644
--- a/arch/arm64/boot/dts/broadcom/Makefile
+++ b/arch/arm64/boot/dts/broadcom/Makefile
@@ -1,6 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 dtb-$(CONFIG_ARCH_BCM2835) += bcm2837-rpi-3-b.dtb
 dtb-$(CONFIG_ARCH_BCM2709) += bcm2710-rpi-3-b.dtb
+dtb-$(CONFIG_ARCH_BCM2835) += bcm2710-rpi-3-b.dtb
 
 subdir-y	+= northstar2
 subdir-y	+= stingray
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1168_ebd8f9b3fd0f96239b2476e9669621e1ee7899ba.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1168_ebd8f9b3fd0f96239b2476e9669621e1ee7899ba.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
